graphstorm-processing/graphstorm_processing/distributed_executor.py

```diff
@@ -503,25 +628,27 @@ def main():
 
     # local output location for metadata files
     if execution_env == ExecutionEnv.SAGEMAKER:
-        local_output_path = "/opt/ml/processing/output"
+        local_metadata_output_path = "/opt/ml/processing/output"
     else:
         if filesystem_type == FilesystemType.LOCAL:
-            local_output_path = gsprocessing_args.output_prefix
+            local_metadata_output_path = gsprocessing_args.output_prefix
         else:
             # Only needed for local execution with S3 data
-            local_output_path = tempdir.name
+            local_metadata_output_path = tempdir.name
```

Comment:
Why change the name?

Reply:
We also have a member var named `self.output_prefix` which is where we put the processed data, and it can be S3 or local. 

We rename this to make it clear that this is a local path intended for metadata files only